src/views/root.jsx

```diff
@@ -0,0 +1,11 @@
+import ResponsiveAppBar from '../components/responsiveAppBar';
+import { Outlet } from "react-router-dom";
+
+export default function TrendingMedia() {
+    return (
+        <div>
+            <ResponsiveAppBar />
+            <Outlet/>
+        </div>
```

Comment:
No need to change, just one thing to keep in mind: if you want two components in your render you can use fragments instead of divs since divs add styling:
```suggestion
        <>
            <ResponsiveAppBar />
            <Outlet/>
        </>
```